torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -229,6 +242,39 @@ class LLVMCodeGenImpl : public IRVisitor {
 
   void optimize(llvm::Module& M);
 };
+
+// TODO: more extensive testing.
+// TODO: benchmarking.
```

Comment:
There are some benchmarks in the PR, do you plan to add something more? If so, it would be helpful to elaborate in the comment (and similarly in the previous comment about more tests).